### PR TITLE
Allow for hvplot returning dynamic layouts

### PR DESCRIPTION
**Pull request by jmosbacher**
This addresses issue #444.

### CODE REVIEW

File: intake/gui/source/defined_plots.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def setup(self):
         self.select = pn.widgets.Select(options=self.options, height=30,
                                         align='center', width=200)
         self.desc = pn.pane.Str()
-        self.pane = pn.pane.HoloViews(self._plot_object(self.selected))
+        self.pane = pn.Column(pn.panel(self._plot_object(self.selected)))
```
Comment (philippjfr) [inline intake/gui/source/defined_plots.py]: Layouts will call pn.panel automatically if needed:

```suggestion
        self.pane = pn.Column(self._plot_object(self.selected))
```

Comment (philippjfr) [reply intake/gui/source/defined_plots.py]: I'd probably also rename `pane` -> `panel` throughout.

**Comment by philippjfr** [comment]
Thanks for these changes. It looks like tests also need to be updated. Otherwise this looks good to me.

**Comment by jmosbacher** [comment]
Yeah sorry I forgot to update the tests. I'll do that asap. Thanks

**Comment by philippjfr** [comment]
@martindurant This seems ready to merge to me.

**Comment by jmosbacher** [comment]
@philippjfr would you suggest already finding a better name than "pane" for the plot display in this PR or leave that for another time? since panel is already taken in this class, i guess plot_preview or plot_display would make sense. Alternatively, i guess i could make a DefinedPlot class with its own panel attribute that also inherits  from BaseView.

**Comment by martindurant** [comment]
Leave it as pane, I think - too many names already. Just so long as we are consistent.

**Pull request merged**